spec/requests/api/v1/sessions_spec.rb

```diff
@@ -1,26 +1,38 @@
 require 'rails_helper'
 
-RSpec.shared_examples 'done session' do
-  let(:in_progress_session) { Session.find(json_response[:id]) }
-
-  it 'sets session status as done' do
-    expect(json_response[:status]).to eq('in-progress')
-    expect {
-      post(api_v1_sessions_path, headers: headers, params: params, as: :json)
-      in_progress_session.reload
-    }.to change(in_progress_session, :status).from('in-progress').to('done')
+RSpec.shared_examples 'no active session' do
+  context 'when it is home session' do
+    let(:params) { { session: { type: 'home' } } }
+
+    it 'should create new home session' do
+      post '/api/v1/sessions/ping', params: params, headers: headers, as: :json
```

Comment:
can we check that an actual session was created in the DB here?

Reply:
Updated! Please have a look!